.changeset/swift-laws-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
```

Comment:
Might be nit-picking, but I _think_ this is probably a minor given how it changes rendering:

```suggestion
'@astrojs/starlight': minor
```